classes/class-ep-debug-bar-elasticpress.php

```diff
@@ -162,12 +166,84 @@ public function render() {
 						<?php endif; ?>
 						<a class="copy-curl" data-request="<?php echo esc_attr( addcslashes( $curl_request, '"' ) ); ?>">Copy cURL Request</a>
 					</li><?php
-					
+
 				endforeach;
 
 			?></ol><?php
 
 		endif;
 	}
 
+	/**
+	 * Display debug info that effects the Elastic Press logs being visible or not.
+	 */
+	protected function display_debugging_info() {
+		$is_debug_errors = false;
+		?>
+		<div class="qm-boxed">
+			<section>
+				<h3><?php esc_html_e( 'Debug Info', 'debug-bar' ); ?></h3>
+				<p>
+					<?php esc_html_e( 'Elastic Press query logs not being displayed can be due to one of the following reasons:', 'debug-bar' ); ?>
+				</p>
+				<table>
+					<tbody>
+						<?php
+						$is_wp_debug = defined( 'WP_DEBUG' ) && WP_DEBUG;
+						$is_wp_ep_debug = defined( 'WP_EP_DEBUG' ) && WP_EP_DEBUG;
+
+						// Display error message only if both WP_DEBUG and WP_EP_DEBUG are turned off.
+						if ( ! ( $is_wp_debug || $is_wp_ep_debug ) ) :
+							$is_debug_errors = true;
+							?>
+						<tr>
+							<td><?php echo '&#10060;'; // cross mark. ?></td>
+							<td>
+								<?php
+								echo wp_kses(
+									sprintf( __( 'Either %s or %s should be defined and set to true', 'debug-bar' ), '<code>WP_DEBUG</code>', '<code>WP_EP_DEBUG</code>' ),
+									[ 'code' => [] ]
+								);
+								?>
+							</td>
+						</tr>
+						<?php
+						endif;
+						?>
+
+						<?php
+						// Data is being currently indexed.
+						if ( ep_is_indexing() || ep_is_indexing_wpcli() ) :
+							$is_debug_errors = true;
+							?>
+						<tr>
+							<td><?php echo '&#10071;'; // exclamation mark. ?></td>
+							<td>
+								<?php esc_html_e( 'Index is currently in sync', 'debug-bar' ); ?>
+							</td>
+						</tr>
+						<?php
+						endif;
+						?>
+
+						<?php
+						// No debug errors to display.
+						if ( ! $is_debug_errors ) :
```

Comment:
With this we're showing some info that isn't really needed, the reason for not showing the info about which constant is set is because when things are working you don't need to change anything. It'll be clear if things need to be changed because you have the message above and the user will be able to see the logs. Lets remove this bit.